wakuv2/waku.go

```diff
@@ -1087,51 +1087,18 @@ func (w *Waku) Start() error {
 
 	go func() {
 		defer w.wg.Done()
-
+		ticker := time.NewTicker(5 * time.Second)
```

Comment:
Can we make it a constant? Also 5 seconds seems pretty frequent.

Reply:
Hmm, previous ticker also used to 5sec, so i just went with that. any suggestions or thoughts as to what can be optimal value?

Reply:
I usually starts with a slower value and make it faster when requires by people. It's probably fine here as you are using the existing ticker.